config/base_config.json

```diff
@@ -3,6 +3,7 @@
   "hash": "",
   "tag": "",
   "apiUrl": "",
+  "metricApiUrl": "",
```

Comment:
There is a dev config that will need a real value. The cypress tests use that to start. The tests are working now because there are no tests that make any calls to this url so it is not throwing any errors.

Reply:
Look at the dev config in `config`